tests/util.go

```diff
@@ -243,7 +243,7 @@ func AccSequence(addr string) uint64 {
 
 func Account(addr string) *sdkAuth.BaseAccount {
 	var bz string
-	_, err := tClient.Call("account", map[string]interface{}{"address": addr, "cdc": true}, &bz)
+	_, err := tClient.Call("account_cdc", map[string]interface{}{"address": addr, "cdc": true}, &bz)
```

Comment:
Please remove the cdc parameter

Reply:
fixed

Reply:
commits at 2b7649aa86e67524978b200edc76d6cd4e842e3e